dictionaries/top-10000-english-words.json

```diff
@@ -4368,7 +4368,7 @@
 "STRAOETS": "streets",
 "TPHAEUGS/WAOEUD": "nationwide",
 "SROERBGT": "vector",
-"TKEFPL": "definitely",
+"TKEFPBLT": "definitely",
```

Comment:
Per discussion on discord, going with `-LT` for consistency with "indefinitely" (`EUPB/TKEFPBLT').